src/terminal/adapter/adaptDispatch.hpp

```diff
@@ -21,8 +21,6 @@ Author(s):
 #include "terminalOutput.hpp"
 #include <math.h>
 
-#define XTERM_COLOR_TABLE_SIZE (256)
```

Comment:
This is duplicate of https://github.com/microsoft/terminal/blob/94fc40ed31c0de050304248644458b21d0cce2e9/src/inc/conattrs.hpp#L71